.github/workflows/build-and-test.yml

```diff
@@ -46,43 +46,61 @@ jobs:
           yarn build
 
       # Execute the tests.
-      - name: Test
+      - name: Run Unit Tests
         run: yarn test
         env:
           # The test result file name can be controlled using the following environment variable.
           JEST_JUNIT_OUTPUT_NAME: unit-test-results-${{ runner.os }}.xml
 
-      # Upload Test Results (The different files for the OSes will end up in the same artifact).
-      - name: Upload Test Results
+      # Upload Unit Test Results (The different files for the OSes will end up in the same artifact).
+      - name: Upload Unit Test Results
         if: always()
         uses: actions/upload-artifact@v3
         with:
           name: unit-test-tesults
           # Include the unit-test-results folders (which is in the root of the workspace).
           path: unit-test-results
+          retention-days: 30
+
+      # Run PlayWright tests, only on Linux container.
+      - name: Install Playwright Browsers
+        if: runner.os == 'Linux'
+        run: yarn --cwd ./e2e-tests/ playwright install --with-deps
+      - name: Run Playwright tests
+        if: runner.os == 'Linux'
+        uses: coactions/setup-xvfb@v1
+        with:
+          run: yarn --cwd ./e2e-tests/ test
```

Comment:
I thin it would be nicer to put that into a script `ui-tests` or something similar within the `package.json` on the root level or the one in the e2e-tests. What do you think?

Reply:
I added a "ui-test" yarn script in the root for this, good point!